source/wp-content/themes/wporg-parent-2021/patterns/heading-with-arrow.php

```diff
@@ -6,12 +6,12 @@
  */
 
 ?>
-<!-- wp:group {"align":"full","backgroundColor":"blueberry-1"} -->
-<div class="wp-block-group alignfull has-blueberry-1-background-color has-background">
-	<!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"spacing":{"padding":{"top":"97px","bottom":"97px"},"blockGap":"0px"}},"className":"wporg-pattern__heading-with-arrow"} -->
-	<div class="wp-block-group alignfull wporg-pattern__heading-with-arrow has-link-color" style="padding-top:97px;padding-bottom:97px">
-		<!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"300","lineHeight":"1.1","letterSpacing":"-0.02em"}},"fontSize":"heading-cta","fontFamily":"inter"} -->
-		<h2 class="has-inter-font-family has-heading-cta-font-size" style="font-style:normal;font-weight:300;letter-spacing:-0.02em;line-height:1.1"><a href="https://wordpress.org/download/">Get started</a></h2>
+<!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"spacing":{"padding":{"top":"var:preset|spacing|60","right":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|60"}}},"backgroundColor":"blueberry-1","textColor":"white","layout":{"inherit":true}} -->
+<div class="wp-block-group alignfull has-white-color has-blueberry-1-background-color has-text-color has-background has-link-color" style="padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--60)">
+	<!-- wp:group {"align":"wide"} -->
```

Comment:
I just realized that since we already have `padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--60)` in parent group, so there's still padding existing after removing this group.

Reply:
Do you think changing from using a variable to an exact number is better?

`"spacing":{"padding":{"top":"80px","right":"80px","bottom":"80px","left":"80px"}}},`
`style="padding-top:80px;padding-right:80px;padding-bottom:80px;padding-left:80px"`

Upside: Users can see padding in GB UI tools.
![image](https://user-images.githubusercontent.com/18050944/183481939-e227b270-e7a9-44e7-9a47-a529e3507521.png)
Downside: We can't configure it through theme.json directly.

Reply:
There are two group containers here so that the top-level one can be full-width across the screen, and the child group is `wide` to give the heading a max-width. This also lets authors put content inside the wide group so it lines up correctly with the heading, like this example:

<img width="2408" alt="Screen Shot 2022-08-08 at 3 27 53 PM" src="https://user-images.githubusercontent.com/541093/183498528-edc8ff3a-30e5-47ec-9d44-ecf53708363c.png">
 
> Do you think changing from using a variable to an exact number is better?

No, because `--wp--preset--spacing--60` is not an exact number, it's [a responsive value using clamp](https://github.com/WordPress/wporg-parent-2021/blob/fb178ec3bce9ef150ed13701ec2db439e6ea6035/source/wp-content/themes/wporg-parent-2021/theme.json#L487-L491). Users can still override this by setting values, but this way the default value adjusts to the screen size. Eventually this method should be supported by the UI too (there's an in-progress PR here: https://github.com/WordPress/gutenberg/pull/42173), so I'd prefer to stick with this.

Reply:
> This also lets authors put content inside the wide group so it lines up correctly with the heading
I see, thanks for the clarification!

>  there's an in-progress PR here: https://github.com/WordPress/gutenberg/pull/42173
👍!